lib/Utils/Git.pm

```diff
@@ -52,7 +60,14 @@ sub git_clone {
     $git_cmd =~ s/git/git -c http.sslVerify=false/ if $args{skip_ssl_verification};
 
     # Checkout branch
-    $git_cmd .= " -b $args{branch}" if $args{branch};
+    $git_cmd .= " --branch $args{branch}" if $args{branch};
```

Comment:
my 2cents for a next PR: manage all this arguments with a list and join it at the end

```
my @git_cmd = ('git');

push @git_cmd,  '-c http.sslVerify=false'  if $args{skip_ssl_verification};
push @git_cmd, 'clone';

...

assert_script_run(joint(' ',@git_cmd));
...


```

Reply:
Yeah sure... 

btw... joint? :D